src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -49,15 +49,15 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
     response.setContentType("application/json");
-
+    //first, check if the user is provided
     String user = request.getParameter("user");
 
     if (user == null || user.equals("")) {
       // Request is invalid, return empty array
       response.getWriter().println("[]");
-      return;
+      return;//stop 
```

Comment:
Nit: you can remove the comment "//stop". It is already obvious what "return" does. This way you reduce clutter in the code.